awx/main/apps.py

```diff
@@ -1,8 +1,17 @@
 from django.apps import AppConfig
+from django.db.models.signals import pre_migrate
 from django.utils.translation import ugettext_lazy as _
 
 
+def raise_migration_flag(**kwargs):
+    from awx.main.tasks import set_migration_flag
+    set_migration_flag()
```

Comment:
You probably actually want `set_migration_flag.apply_async()` so that it gets broadcast everywhere, right?